### ISSUE TITLE
Hypoplastic eyelid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7522]
**Preferred term label:**
Hypoplastic eyelid

**Synonyms**

**Definition (free text, please give PubMed ID)**
Underdevelopment of the eyelid.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
200110 
PMID: 26119818  3/10
 209885 3/12 (same paper)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
term already existed, adding synonym
